config/src/config/storage_config.rs

```diff
@@ -136,6 +139,7 @@ impl StoragePrunerConfig {
             ledger_prune_window: ledger_store_prune_window,
             ledger_pruning_batch_size,
             state_store_pruning_batch_size,
+            user_pruning_window_offset: ledger_store_prune_window / 5,
```

Comment:
Too large? (we are talking about making the ledger window a few hundred millions). I think let's just do something like 100k.

Reply:
This is dead code?  I just removed it.

And I changed the amount to 200k.  You can adjust the numbers later